sql/moz-fx-data-shared-prod/monitoring_derived/outerbounds_flow_description_v1/metadata.yaml

```
@@ -11,3 +11,5 @@ external_data:
     - https://docs.google.com/spreadsheets/d/1E0kDpHwwtDnkMxAXbeEIzMTajckSHWJ2swF39JdRR1g # URL to the spreadsheet
   options:
     skip_leading_rows: 1 # number of rows that should be skipped, e.g if there are header rows
+scheduling:
+  dag_name: bqetl_monitoring
```

```suggestion
  dag_name: bqetl_monitoring
  date_partition_parameter: null
```
`date_partition_parameter` set to `null` will ensure that the entire table is overridden, not just a partition